zio/zngio/writer.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // DefaultLZ4BlockSize is a reasonable default for WriterOpts.LZ4BlockSize.
-const DefaultLZ4BlockSize = 16 * 1024
+const DefaultLZ4BlockSize = 500 * 1024
```

Comment:
No objection here—just curious—but why not 512?

Reply:
No reason